embedded-graphics/src/lib.rs

```diff
@@ -113,6 +113,8 @@
 //! // Create a draw target using the builtin MockDisplay. In real applications this would be
 //! // replaced by a draw target that is provided by a display driver crate.
 //! let mut display = MockDisplay::default();
+//! display.set_allow_overdraw(true);
+//! display.set_allow_out_of_bounds_drawing(true);
```

Comment:
Should these be hidden in this example? If not, a oneline comment explaining why they're present would be useful.

Reply:
The `MockDisplay` creation was unhidden, when `lib.rs` and `README.md` shared the same example. But if this example is only used in `lib.rs` I would suggest to also hide the `MockDisplay::default()` line and replace that by a comment that `display` is the `DrawTarget` impl from a driver crate. OK?

Reply:
Yep, that sounds good to me

Reply:
Is the comment I've added OK?

Reply:
Yep that looks good, thanks